docs/graphql/core/deployment/deployment-guides/nhost-one-click.rst

```diff
@@ -25,36 +25,31 @@ One-click deploy on Nhost
 
 Once you're logged into your Nhost account, click the button bellow to configure your project.
 
-.. image:: https://nhost.io/images/create-project-button.png
-   :width: 300px
-   :class: no-shadow
-   :alt: Deploy with Nhost
+.. thumbnail:: /img/graphql/core/deployment/create-project-nhost.png
```

Comment:
For me, the create project button looks like this 🤔 

![image](https://user-images.githubusercontent.com/5722022/97150278-2df7fb80-176e-11eb-9741-3cb4e0e0f18b.png)

Reply:
On the first creation of an account, the former image is what is there. But I can upgrade it to this. :)